### PR TITLE
chore: Move website event from release to pre-release

### PR DESCRIPTION
**Pull request by ebebbington**

### CODE REVIEW

**Comment by ebebbington** [comment]
Ok deffo ready for review

File: .github/workflows/pre_release.yml [hunk 1]
```diff
@@ -35,6 +35,10 @@ jobs:
       - name: Bump Versions
         run: deno run -A ./console/bumper_ci_service.ts --version=${{ github.ref }} # vrsion passed so we know what the new release version will be
 
+      - name: Send Event to Website to Update Versions
+        run: |
+          curl -XPOST -u "${{ secrets.CI_USER_NAME }}:${{ secrets.CI_USER_PAT }}" -H "Accept: application/vnd.github.everest-preview+json" -H "Content-Type: application/json" https://api.github.com/repos/drashland/website/dispatches --data '{"event_type": "update_docs", "client_payload": { "module_to_update": "drash", "version": "${{ github.ref }}" }}'
```
Comment (crookse) [inline .github/workflows/pre_release.yml:40]: can we break this apart into separate lines 😄

Comment (crookse) [reply .github/workflows/pre_release.yml:40]: nvm. idk if breaking it apart into separate lines would work the first time and i want this to work the first time. resolving.

**Comment by ebebbington** [comment]
PR looks good to you eric?

**Review by crookse — APPROVED** [review]

**Comment by ebebbington** [comment]
im just gonna remove the logic that deletes the remote branch

**Review by crookse — APPROVED** [review]

**Pull request merged**